docs/src/pages/components/slider.mdx

```diff
@@ -91,48 +98,56 @@ import { Slider } from '@marigold/components';
 
 ### Width
 
+Here you can see a `<Slider>` with a smaller width than the default.
+
 ```tsx preview file=components/Slider/slider-width.demo.tsx
 
 ```
 
 ### Disabled
 
+You can also disable the `<Slider>` if you want user not to select a value.
+
 ```tsx preview file=components/Slider/slider-disabled.demo.tsx
 
 ```
 
 ### Step
 
+This example shows how to use the `step` property. The value is set in steps of 10.
+
 ```tsx preview file=components/Slider/slider-steps.demo.tsx
 
 ```
 
 ### Max value
 
+If you want to use a other `maxValue` than 100. You can set the property and change it to the value you want.
+
 ```tsx preview file=components/Slider/slider-max.demo.tsx
 
 ```
 
 ### Value (controlled)
 
+This example shows how to build a controlled `<Slider>`. You see that there is used the `value` prop with an `onChange`. The default value in this case is set to 75, but if you change the value will be updated. If you want an uncontrolled `<Slider>` see next example.
```

Comment:
```suggestion
This example shows how to build a controlled `<Slider>`. The `value` prop together with the `onChange` prop are used to update and set the current value. The default value in the example is set to "75", if you manually change the value it will be updated. If you want an uncontrolled `<Slider>` see the next example.
```